draft-ietf-webpush-protocol.xml

```diff
@@ -1084,7 +1084,7 @@ HEADERS      [stream 82] +END_STREAM
             A push service MAY expire a subscription set at any time which MUST also
             expire all push message subscriptions in the set. If a user agent has an
             outstanding request to a push subscription set (<xref target="monitor-set"/>)
-            this can be signaled by returning a 400-series status code, such as 410 (Gone).
+            this can be signaled by returning a 410 (Gone) status code.

```

Comment:
We can't use 410 in multiple places?  It's not like the subscription (set) was not found, it is just that it disappeared while the response was being generated.